src/index.html

```diff
@@ -10,7 +11,32 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="main">
+      <form class="search__bar content__form"
+      method="get"
+      data-qa="big">
```

Comment:
code it not aligned 
<img width="1239" alt="Screenshot 2023-10-18 at 13 15 10" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64746943/663c2f6d-ec4c-40d6-85ba-e52f5c0501b0">